scala-cli-src

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 set -euo pipefail
 SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
-LAUNCHER="$(cd "$SCRIPT_DIR" && ./mill show cli.launcher </dev/null | jq -r . | sed 's/ref:[a-z0-9]*://')"
+LAUNCHER="$(cd "$SCRIPT_DIR" && ./mill show cli.launcher </dev/null | jq -r . | sed 's/ref:[a-z0-9]*:[a-z0-9]*://')"
```

Comment:
?

Reply:
right... `scala-cli-src` was broken somewhere along the way, since `./mill show cli.launcher` output changed.
ok.